syntax/NeogitStatus.vim

```diff
@@ -49,13 +54,14 @@ for section in b:sections
   execute 'syn region Neogit' . id . 'Region start=/^' . section . '\ze.*/ end=/./ contains=Neogit' . id
 endfor
 
-syn region NeogitHeadRegion         start=/^Head: \zs/        end=/$/ contains=NeogitBranch
```

Comment:
Any reason for the spacing change?

Reply:
I originally indented it to fit `NeogitHeadDetachedRegion` before renaming to `NeogitDetachedRegion` as it was a little long.

Anyhow, it is resolved now.

I always get the same kind of feeling of *oh* when I encounterd old vimscript and need to remember to escape my strings properly as they aren't quoted. The good ol' backslash because you wanted a space in an option, or a combintation of colons because vim does not support lists for options.

Reply:
Yeah... I don't miss vimscript. These patterns especially, for syntax regions/matches. I've done a lot of spelunking in the gitrebase syntax file for inspiration, but oh man, lua/extmarks is a much easier API